EcoSoapBank/Helpers/Colors.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Colors.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 8/10/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+    static let esbGreen = UIColor(named: "ESBGreen")!
+    static let downyBlue = UIColor(named: "DownyBlue")!
+    static let codGrey = UIColor(named: "CodGrey")!
+    static let montanaGrey = UIColor(named: "MontanaGrey")!
+    static let shuttleGrey = UIColor(named: "ShuttleGrey")!
+    static let silver = UIColor(named: "Silver")!
```

Comment:
Nice names.

Reply:
Haha, I used a site that names any hex color.